test/controllers/customers_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class CustomersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @customer = customers(:one)
+  end
+
+  test "should get index" do
+    get customers_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_customer_url
+    assert_response :success
+  end
+
+  test "should create customer" do
+    assert_difference('Customer.count') do
+      post customers_url, params: { customer: { email: @customer.email, first_name: @customer.first_name, last_name: @customer.last_name, phone: @customer.phone } }
+    end
```

Comment:
This test did not pass from my end. Please make appropriate changes to approve this branch. I believe it has to do with the post path that you provided.